apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/EphemeralTransaction.cs

```diff
@@ -108,8 +108,9 @@ async Task<IEnumerable<long>> Impl()
         /// <param name="mutations">The list of changes to apply.</param>
         /// <param name="cancellationToken">A cancellation token used for this task.</param>
         /// <param name="timeoutSeconds">The timeout which will apply to the commit part of this method.</param>
+        /// <param name="priority">The priority that will be used for the commit request.</param>
         /// <returns>The number of rows modified.</returns>
-        public Task<int> ExecuteMutationsAsync(List<Mutation> mutations, CancellationToken cancellationToken, int timeoutSeconds)
+        public Task<int> ExecuteMutationsAsync(List<Mutation> mutations, CancellationToken cancellationToken, int timeoutSeconds, Priority priority)
```

Comment:
Note to self: this is an internal class, so it's not a breaking change despite looking like one. Ditto the ISpannerTransaction change.

Reply:
(Not relevant anymore, as with the latest changes there are no changes to the public methods of `EphemeralTransaction` or `ISpannerTransaction`)